README.md

```diff
@@ -76,8 +76,6 @@ You should get a result similar to below.
 ```
 NAME                                               READY   STATUS      RESTARTS   AGE
 elasticsearch-master-0                             1/1     Running     0          62m
-elasticsearch-master-1                             1/1     Running     0          62m
-elasticsearch-master-2                             1/1     Running     0          62m
```

Comment:
I see `elasticsearch-master-0` and `elasticsearch-master-1` running on the stg cluster. Are they supposed to be up and running?

Reply:
Yes. They are supposed to be up and running 3 replicas in production and 2 replicas in staging.